### ISSUE TITLE
Recommendations (door hanger)

### ISSUE THREAD

**Issue by Osmose** [mozilla/normandy#329]
This issue covers implementing the first iteration of @raymak's contextual feature recommendation as an action. 

**Blocked by:**
- [ ] UI Treatment study to determine the UX for how we want to present recommendations.

## Notes
- Possibly wait for UI form testing to see if door hanger is ready
- Need to meet with kamyar about minimum contextual triggers
- Need to figure out how to not accidentally identify users by getting results of a recommendation
  - Going to meet with glind about ways to avoid this
- Need to handle rate-limiting and queuing recommendations so users are not overloaded.

**Comment by mythmon** [comment]
Contextual feature recommendation is something that has largely been taken over in other projects, especially opt-out and opt-in add-on studies. Normandy's declarative model never grew into something rich enough to represent CFR, and I don't think it should anymore.

Closing this as WONTFIX.
